lisa/tools/mv.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Mv(Tool):
+    @property
+    def command(self) -> str:
+        return "mv"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
```

Comment:
Can you also add the support to move folders?

Reply:
The mv command will move folders with the current design. It does not require anything special. I will go ahead and rename to just move.